docs/contributor/benchmarks/istio-access-logs-otlp-provider.md

```diff
@@ -1,11 +1,11 @@
 # Istio Access Logs OTLP Provider Load Test
-Goal: Ensure that enabling Istio access logs via the new OTLP provider `kyma-logs` does not negatively impact Envoy's performance.
+Goal: Ensure that enabling Istio access logs with the new OTLP provider `kyma-logs` does not negatively impact Envoy's performance.
 
 ## Setup
 
 ### Base Setup
 1. `istio` module deployed
-2. Change `kyma-system/istio-controller-manager` Deployment image to: `europe-central2-docker.pkg.dev/sap-se-cx-kyma-goat/istio/istio-manager:PR-1374`
+2. Change `kyma-system/istio-controller-manager` Deployment image to: `europe-central2-docker.pkg.dev/sap-se-cx-kyma-goat/istio/istio-manager:PR-1374` (Alternatively, scale down the `istio-controller-manager` Deployment and apply [istio-config.yaml](./assets/istio-access-logs/istio-config.yaml) for additional, custom provider configurations)
```

Comment:
Does this mean the alternative method provides more config options than the first method? In that case, I'd suggest

```suggestion
2. Change `kyma-system/istio-controller-manager` Deployment image to: `europe-central2-docker.pkg.dev/sap-se-cx-kyma-goat/istio/istio-manager:PR-1374`.
Alternatively, if you need additional, custom provider configurations, scale down the `istio-controller-manager` Deployment and apply [istio-config.yaml](./assets/istio-access-logs/istio-config.yaml).
```